modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/PagesWriteThrottleSandboxTest.java

```diff
@@ -172,14 +194,27 @@ public void testThrottle() throws Exception {
                 }
             }, "metrics-view");
 
+            final boolean sawForm = false;
```

Comment:
It is always `false`, because it is a manual sandbox, not a test for being run automatically. The user can turn it on and see how it goes in this mode :)

`saw form` is actually about the traffic having a form of a saw: for 10 seconds we add data at full speed, then for 10 seconds we don't do anything, then we repeat step 1.

I renamed the variable to `intermittentPutsMode`, does it look better, what do you think?